src/OrbitBase/TemporaryFile.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitBase/TemporaryFile.h"
+
+#include <stdlib.h>
+
+namespace orbit_base {
+
+void TemporaryFile::Create() noexcept {
+  std::error_code error;
+  std::filesystem::path temporary_dir = std::filesystem::temp_directory_path(error);
+  std::string file_path = (temporary_dir / "orbit_XXXXXX").string();
```

Comment:
The man page and the examples I saw all use capital X's. Since the tests work this doesn't seem to matter though.

Reply:
aren't we also using capital X's? Not sure I understand what you mean here.

Reply:
Wow, these look like small x's - I did not even bother to copy them over.  You are right, these are capital letters.